### DISCUSSION TITLE
Contribution widget in iOS

### DISCUSSION THREAD

**Discussion by PolNun** [community/community#33609] [Mobile]
Contribution squares green colors are the opposite of the green colors shown on the website.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.80.0 (78250505)
Device: iPhone 6S Plus (iOS 15.7)
</details>

**Comment by eliperkins** [comment]
Hey @PolNun , thanks for this feedback! I think I'm able to reproduce this issue myself, so I've put it together in an internal issue. I should have a fix for this bug out shortly, and I'll follow up with you once it's addressed.
